__init__.py

```diff
@@ -1150,6 +1151,38 @@ def _speak_weather(self, dialog):
         self.log.info("Speaking dialog: " + dialog.name)
         self.speak_dialog(dialog.name, dialog.data, wait=True)
 
+    @skill_api_method
+    def get_current_weather(self, message=None):
+        """Get the current temperature and condition code.
+        
+        Args:
+            message (Message): [Optional] mock message will be parsed to
```

Comment:
The rest of the code in this skill uses Python type hints instead of docstring type hints.  Why do it differently here?

Also, you are getting the current weather.  What would a test need to pass into this method?  You could even rename the method to be `get_current_local_weather()`.

The use case here is to put local weather on the Home Screen.  Why write it to be any broader than that?

Reply:
moving to the `get_weather_for_coordinates` removes any need for a message object so have removed it entirely